core/controller.go

```diff
@@ -216,13 +230,34 @@ func (c *Controller) ListStreamHandler(w http.ResponseWriter, r *http.Request, _
 		return
 	}
 	dto := []*SummariseDTO{}
+
+	// active streams
 	for key, stream := range c.streams {
+		aliasname := ""
```

Comment:
Can we camelcase this please to `aliasName`? Makes the code more consistent and easier to read

Reply:
Yes, I'll do that now